### DISCUSSION TITLE
PP-OCRv5_mobile_rec_pretrained Model SFT - LOSS SET

### DISCUSSION THREAD

**Discussion by WangM1125** [PaddlePaddle/PaddleOCR#18311] [General]
English **Question:**My training task is to read numbers. The diversity of the dataset has led to poor model training results. The originally simple model with black background and red text achieved good results after fine-tuning, but when blue text was added later, the performance became worse. I want to optimize the parameters. I learned that the loss value of this model is divided into two parts. The most important part of my task is CTCLOSS, so I want to abandon the loss of NRTRLOSS, which has an impact on my backpropagation. However, I found that it seems impossible to achieve this. I set the parameters like this, but it seems that it didn't take effect. Are there any other solutions? The solution given by GPT is for me to modify the original code of paddle.

问题：我的训练任务是读数，数据集的多样性导致模型训练结果变差，原本单一的黑屏红字的数据微调的模型效果不错，后面又累加了蓝色字体的，就不太行了，我想优化参数，了解到这个模型的损失值分为两个部分，我这个任务最重要的是CTCLOSS，所以想撇弃NRTRLOSS这个损失对我的反向传播造成的影响，但是我发现好像实现不了，我是这样设置参数的，好像没有生效，还有啥别的办法吗？GPT给出的方案是要我改动paddle原码

Loss:
  name: MultiLoss
  loss_config_list:
    - CTCLoss:
        weight: 1.0
    - NRTRLoss:
        weight: 0.0
